docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -53,6 +53,39 @@ They receive the pre-processed auction instance from the driver sub-component an
 
 The main flow of arbitrating an auction end-to-end can be seen in the following sequence diagram:
 
+```mermaid
+sequenceDiagram
+    actor User
+    participant orderbook
+    participant autopilot
+    participant solver
+
+    User->>orderbook: /post_order
+    autopilot->>orderbook: Request solvable orders
+    activate autopilot
+    activate orderbook
+    orderbook->>autopilot: Response solvable orders
+    deactivate orderbook
+    Note over autopilot: Create auction
+    autopilot->>solver: /solve
+    activate solver
+    solver-->>autopilot: Solution
```

Comment:
We should probably use a different word than `solution` because the next call fetches the "Solution with calldata" which begs the question why we split it into 2 calls in the first place.
```suggestion
    solver-->>autopilot: Score
```